package/build-wheels-macos.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+set -x -e
+
+function retry {
+  local retries=$1
+  shift
+
+  local count=0
+  until "$@"; do
+    exit=$?
+    wait=$((2 ** $count))
+    count=$(($count + 1))
+    if [ $count -lt $retries ]; then
+      echo "Retry $count/$retries exited $exit, retrying in $wait seconds..."
+      sleep $wait
+    else
+      echo "Retry $count/$retries exited $exit, no more retries left."
+      return $exit
+    fi
+  done
+  return 0
+}
+
+brew update
+brew upgrade
+brew install wget python cmake || true
+# TODO: try without brew install of boost, but use version below
+brew install git cmake boost eigen freeimage glog gflags suite-sparse ceres-solver glew cgal qt5
+
+brew install llvm libomp
+
+brew info gcc
+brew upgrade gcc
+brew info gcc
+
+echo 'export PATH="/usr/local/opt/qt@5/bin:$PATH"' >> /Users/runner/.bash_profile
+export Qt5_CMAKE_DIR="/usr/local/opt/qt@5/lib/cmake/Qt5"
+
+CURRDIR=$(pwd)
+ls -ltrh $CURRDIR
+
+# Build Boost staticly
+mkdir -p boost_build
+cd boost_build
+retry 3 wget https://boostorg.jfrog.io/artifactory/main/release/1.73.0/source/boost_1_73_0.tar.gz
+tar xzf boost_1_73_0.tar.gz
+cd boost_1_73_0
+./bootstrap.sh --prefix=$CURRDIR/boost_install --with-libraries=serialization,filesystem,thread,system,atomic,date_time,timer,chrono,program_options,regex clang-darwin
+./b2 -j$(sysctl -n hw.logicalcpu) cxxflags="-fPIC" runtime-link=static variant=release link=static install
+
+echo "CURRDIR is: ${CURRDIR}"
+
+cd $CURRDIR
+mkdir -p $CURRDIR/wheelhouse_unrepaired
+mkdir -p $CURRDIR/wheelhouse
+
+ORIGPATH=$PATH
+
+PYTHON_LIBRARY=$(cd $(dirname $0); pwd)/libpython-not-needed-symbols-exported-by-interpreter
+touch ${PYTHON_LIBRARY}
+
+declare -a PYTHON_VERS=( $1 )
+
+# Get the python version numbers only by splitting the string
+split_array=(${PYTHON_VERS//@/ })
+VERSION_NUMBER=${split_array[1]}
+
+git clone https://github.com/colmap/colmap.git
+
+sed -i -e 's/Qt5 5.4/Qt5 5.15.2/g' colmap/CMakeLists.txt
+
+for compiler in cc c++ gcc g++ clang clang++
+do
+    which $compiler
+    $compiler --version
+done
+
+echo "Python versions: ${PYTHON_VERS[@]}"
+# Compile wheels
+for PYVER in ${PYTHON_VERS[@]}; do
```

Comment:
@johnwlambert Why do we need to loop over versions here? The CI calls this script for each different value of `pyversion` so `PYTHON_VERS` is always a single version string, not a list.

Reply:
good catch -- yes the loop is a no-op and can be removed